### PR TITLE
Add Vscode Github plugin

### PR DESCRIPTION
**Pull request by vinokurig**
https://github.com/eclipse/che/issues/11867

### CODE REVIEW

File: v3/plugins/ms-vscode/vscode-git/latest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+name: vscode-git
+version: latest
+type: VS Code extension
+displayName: Git
+title: This extension is bundled with Visual Studio Code.
+description: Git SCM Integration.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/microsoft/vscode/tree/master/extensions/git
+category: Other
+firstPublicationDate: '2016-03-11'
```
Comment (benoitf) [inline v3/plugins/ms-vscode/vscode-git/latest/meta.yaml]: hello, AFAIK it's the date published on this registry no ?

File: v3/plugins/ms-vscode/vscode-github-pullrequest/latest/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+name: vscode-github-pullrequest
+version: latest
+type: VS Code extension
+displayName: GitHub Pull Requests
+title: Pull Request Provider for GitHub.
+description: This extension allows you to review and manage GitHub pull requests.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/microsoft/vscode-pull-request-github
+category: Other
+firstPublicationDate: '2018-06-27'
```
Comment (benoitf) [inline v3/plugins/ms-vscode/vscode-github-pullrequest/latest/meta.yaml]: same small remark there

Comment (vinokurig) [reply v3/plugins/ms-vscode/vscode-github-pullrequest/latest/meta.yaml]: will fix that to the date of merge.

File: v3/plugins/ms-vscode/vscode-git/latest/meta.yaml (hunk 1 above)
Comment (vinokurig) [reply v3/plugins/ms-vscode/vscode-git/latest/meta.yaml]: I thought that it is the first commit date of the related repository, will fix that to the date of merge.

**Comment by benoitf** [comment]
@vinokurig could you please also reference https://github.com/eclipse/che/issues/11867 in the description so we're sure it's part of 'must fix' PR

**Comment by azatsarynnyy** [comment]
These two plugins have `latest` versions without the corresponded particular versions (e.g. 1.0.0).
As I understand, `latest` should always correspond to some particular plugin version (copying it's `meta.yaml`).
Should the PR also add the concrete plugins' versions?

**Comment by vinokurig** [comment]
@azatsarynnyy Fixed

**Comment by benoitf** [comment]
Hello, I'm thinking as git is quite a "default" extension, it should be included in the default che-theia image and not be part of the registry

**Comment by benoitf** [comment]
could be hooked in https://github.com/eclipse/che-theia/blob/master/dockerfiles/theia/Dockerfile#L85-L86 and https://github.com/eclipse/che-theia/blob/master/dockerfiles/theia/Dockerfile#L108

**Comment by vinokurig** [comment]
@benoitf 
> Hello, I'm thinking as git is quite a "default" extension, it should be included in the default che-theia image and not be part of the registry

The VsCode git plugin is built from vscode build scripts that are located in the vscode repository, can we include the `vsix` file to the che-theia build?

**Comment by benoitf** [comment]
@vinokurig AFAIK yes
you just use curl or wget to grab it from https://github.com/che-incubator/vscode-git/releases

**Comment by vinokurig** [comment]
@benoitf
* Added the vscode git plugin to che-theia (https://github.com/eclipse/che-theia/pull/328)
* Updated the PR (removed the git extension from plugins)

**Comment by benoitf** [comment]
@vinokurig thanks

**Pull request merged**